syntax = "proto2";

package TokenContainer;

message TokenContainer {
  optional string SessionToken = 1;
  optional string WwnAccessToken = 2;
  optional string ServiceAccessKey = 3;
  required string OliveToken = 4;
}
